spec/models/school_spec.rb

```diff
@@ -12,5 +12,7 @@
 
     it { is_expected.to have_one(:partnership) }
     it { is_expected.to have_one(:lead_provider).through(:partnership) }
+    it { is_expected.to have_and_belong_to_many(:induction_coordinator_profiles) }
+    it { is_expected.to have_and_belong_to_many(:school_domains) }
```

Comment:
This just looks wrong to me. I don't see why we should have a school with many domains. Even if it did, there'd only be a handful and the data wouldn't change that often we could store it in an array field

Reply:
We can always change this, but this was motivated by the idea that you might have an "@multi-academy-trust.com" type email shared by multiple schools